packages/components/ContextualMenu/src/ContextualMenu.tsx

```diff
@@ -58,22 +57,23 @@ export const ContextualMenu = compose<ContextualMenuType>({
 
     const styleProps = useStyling(userProps, (override: string) => state[override] || userProps[override]);
 
-    const containerPropsWin32: IViewProps = {
-      accessible: shouldFocusOnContainer,
-      focusable: shouldFocusOnContainer && containerFocus,
-      onBlur: toggleContainerFocus,
-      style: { maxHeight: maxHeight, width: maxWidth },
-    };
-
     const slotProps = mergeSettings<ContextualMenuSlotProps>(styleProps, {
       root: {
         accessibilityRole: 'menu',
         setInitialFocus: shouldFocusOnMount,
         ...rest,
       },
       container: Platform.select({
-        macos: {},
-        default: containerPropsWin32,
+        macos: {
```

Comment:
`shouldFocusOnContainer` is a win32 prop that all of the container props depended on. I never implemented it for macOS, so it felt wrong to pass these props down for macOS. For ContextualMenu specifically, I think `shouldFocusOnContainer` will always be false, because "menus" don't get focus on native macOS menus, only the menu items. 

maxHeight and maxWidth I initially opted out because those were needed for adding a scrollView to the menu. This caused some side effect where setting maxHeight/width also set minHeight/Width and all the menus looked off, so I excluded it. Apparently that same issue happened on win32 according to @lenahong so 🤷🏾.

I think I need to more fully test the bidirectional nature of callout lifecycle management, but the answer should be yes. If Callout is dismissed on the native side, it calls the JS callback "onDismiss".

Reply:
I thought when we talked you said that the first 'down' arrow after opening the menu highlights the first menu item; where would you consider focus at that point in time at initial open?

Reply:
That's how it works with native menus (NSMenus). I have no idea where NSMenu is putting focus at that point, and I don't have a method to check :') 

I was gonna differ from that behavior and just put focus on the first menu item because that felt more achievable with React Native...